Library/Formula/strider.rb

```diff
@@ -0,0 +1,93 @@
+class UniversalNpm < Requirement
+  fatal true
+  satisfy { which("npm") }
+  def message
+    "npm is required. If you have installed node with `--without-npm` option, reinstall with `--with-npm`."
+  end
+end
+
+class UniversalNode < Requirement
```

Comment:
We haven't really supported this thus far, because until very recently there has only been one Node. I think ` depends_on "node" ` would suffice here, probably.